crates/nu-cmd-extra/src/extra/bits/shift_right.rs

```diff
@@ -56,29 +77,24 @@ impl Command for BitsShr {
         input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
         let head = call.head;
-        let bits: usize = call.req(engine_state, stack, 0)?;
+        let bits: i64 = call.req(engine_state, stack, 0)?;
```

Comment:
Can you tell me the reason to allow negative number input?  I saw you convert it to u32 on line 135 .

Reply:
didn't know how the FromValue implementation for usize handles negative numbers, I just read it and see that it errors out which makes sense. will change this back to usize